packages/patrol_cli/lib/src/commands/build_android.dart

```diff
@@ -42,6 +42,14 @@ class BuildAndroidCommand extends PatrolCommand {
     usesUninstallOption();
 
     usesAndroidOptions();
+    argParser.addOption(
+      'build-number',
+      help: 'A build number to be used as --build-number in Flutter build.',
+    );
+    argParser.addOption(
+      'build-name',
+      help: 'A build name to be used as --build-name in Flutter build.',
+    );
```

Comment:
Consider moving those two into `usesBuildVersion()`(or use some other name) method instead of pasting it multiple times.